### PR TITLE
fix: SFDX:Rename Component now also rename the test file that has the same name as component under __tests__ folder for LWC components

### PR DESCRIPTION
**Pull request by floralan**
### What does this PR do?
Enable SFDX: Rename Component command to rename test file under __tests__ folder for LWC components. Only the test file that has the same name as component will be renamed. 

### What issues does this PR fix or reference?
@W-10976932@

### Functionality Before
SFDX: Rename Component does not rename test file under __tests__ folder in LWC components. 

### Functionality After
SFDX: Rename Component now rename the test file in LWC components as well.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRenameLightningComponent.test.ts [hunk 1]
```diff
@@ -39,7 +41,7 @@ describe('Force Rename Lightning Component', () => {
       readdirStub
         .onFirstCall().resolves([])
         .onSecondCall().resolves([])
-        .onThirdCall().resolves([itemsInHero[1]]);
+        .onThirdCall().resolves([itemsInHero[0]]);
```
Comment (gbockus-sf) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRenameLightningComponent.test.ts:44]: did this change for any particular reason?

Comment (gbockus-sf) [reply packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRenameLightningComponent.test.ts:44]: oh I see... moving the __tests__ got it

**Review by gbockus-sf — APPROVED** [review]
good stuff thanks for the quick turn around 👍

**Comment by gbockus-sf** [comment]
QE:

**Review by gbockus-sf — APPROVED** [review]

**Comment by floralan** [comment]
> 



> 
"isDuplicate" function only check for the existing LWC and Aura Components name aka the folder names. If we check every file name under LWC or Aura for duplicate, when the number of components is large, "rename component" command can be slow. But we can try to implement it and test for the execution time to see if it's acceptable.

**Comment by gbockus-sf** [comment]
> ause that "isDuplicate" function only check for the existing LWC and Aura Components name aka the folder names. If we check every file name under LWC or Aura for duplicate, when the number of components is large, "rename component" command can be slow. But we can try to implement it and test for the execution time to see if it's acceptable.

@floralan I don't think it's a blocker for this PR.  We can discuss the issue and see if it makes sense to worry about it. It would be good to add your above comment to the issue I created.

**Comment by floralan** [comment]
> 

@gbockus-sf  Sounds great! I'll get this PR merged and discuss the issue in the charter of the ticket.

**Pull request merged**